ravenpy/models/rv.py

```diff
@@ -712,84 +712,80 @@ def txt_basin_state(self):
 
 
 class RVH(RV):
```

Comment:
If we don't need special setters and getters, these can be dumb class attributes, no?

Reply:
Some getters implement special twists though:

https://github.com/CSHS-CWRA/RavenPy/blob/feature/channel-profile-rvp-support/ravenpy/models/rv.py#L732-L742

Reply:
Right, looked too quickly.